### PR TITLE
[Add profile] -> [Oludamola Onarinde]

### PR DESCRIPTION
**Pull request by damtrix**
Me i don't understand this PR ooh?
Please add my name to the contributor's list

### CODE REVIEW

File: Contributors.md [hunk 1]
```diff
@@ -108,3 +108,16 @@
 </td>
 <!-- End of column-9 -->
 </table>
```
Comment (Evavic44) [inline Contributors.md:110]: Do you have any questions about contributing? I can help

Meanwhile can you please add your profile inside the `<table></table>` **tag** below 
`<!-- End of column-9 -->`

E.g
```html
<table>
<!-- End of column-9 -->
<!-- Start of column-10 -->
<td align="center">
  <a href="https://github.com/damtrix">
    <img
      src="https://avatars.githubusercontent.com/u/42788669?s=400&u=be1f9d656fa29d9a56bd2d47c19936a550d11532&v=4"
      width="100px"
    />
    <br />
    <sub>Oludamola Onarinde</sub>
  </a>
</td>
<!-- End of column-10 -->

</table>
```

**Review by Evavic44 — CHANGES_REQUESTED** [review]

**Review by damtrix — COMMENTED** [review]
Please help check it seems i have rectify it. thanks

**Comment by Evavic44** [comment]
Hey, @damtrix you have some merge conflicts which means some new changes have been added that you haven't pulled.

To fix that, pull the new changes and then push again. 🙂

**Review by Evavic44 — APPROVED** [review]
Nice work fixing the conflicts man. 🙌🏾🤩 Perfetc!

**Pull request merged**